scripts/developer/developer-install-with-env-and-extras.bat

```diff
@@ -0,0 +1,13 @@
+call conda deactivate
+conda info --envs | grep "pmp"
+if %ERRORLEVEL% EQU 0 (call conda env remove -y --name pmp)
+conda create -y --name pmp python=3.7 &&^
+conda activate pmp &&^
+conda install -y nbstripout &&^
+nbstripout --install &&^
+nbstripout --is-installed &&^
+conda install -y pymedphys --only-deps &&^
+yarn &&^
+pip install -e .[docs] &&^
+pip install -e .[testing] &&^
+pip install -e .[formatting]
```

Comment:
What do you think about making these into [npm scripts](https://docs.npmjs.com/misc/scripts) that run with yarn in a cross-platform way?